test/spec/support/compile-config-spec.js

```diff
@@ -95,9 +97,12 @@ describe('support/compile-config', function() {
             throw new Error('not found: ' + path);
           },
           function __resolve(path) {
+            if (path === 'bpmnlint-plugin-foreign/package.json') {
```

Comment:
As this is unit tests and we are mocking all things I'd rather not depend on the current OS. Couldn't we simply create two test cases, on testing Windows and one testing Unix resolution?

Reply:
Well, the problem is that the resolution is done based on the OS. We're not explicitly using `win32` or `posix`. So we'd have to be able to configure this for the test case.

Reply:
✅